modules/profiles/modules.php

```diff
@@ -133,7 +133,16 @@ public function process() {
             $profile['id'] = $form['profile_id'];
             Hm_Profiles::edit($form['profile_id'], $profile);
         } else {
+            $profiles = $this->get('profiles');
+
+            foreach ($profiles as $existing_profile) {
+                if ($existing_profile["address"] === $profile["address"]) {
```

Comment:
We must have smtp_id in this condition. The idea is to check the address but considering that we can have same adresses(emails) on a different smtp.